cmd/notation/internal/policy/import.go

```diff
@@ -0,0 +1,102 @@
+// Copyright The Notary Project Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package policy
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/notaryproject/notation-go/dir"
+	"github.com/notaryproject/notation-go/verifier/trustpolicy"
+	"github.com/notaryproject/notation/cmd/notation/internal/cmdutil"
+	"github.com/notaryproject/notation/internal/osutil"
+)
+
+type policy interface {
+	Validate() error
+}
+
+// Import imports trust policy configuration from a JSON file.
+//
+// - If force is true, it will override the existing trust policy configuration
+// without prompting.
+// - If isOciPolicy is true, it will import OCI trust policy configuration.
+// Otherwise, it will import blob trust policy configuration.
+func Import(filePath string, force, isOCIPolicy bool) error {
```

Comment:
There are too many `if` statements for `isOCIPolicy`. Consider refactoring this method to two methods.

Reply:
Yes, I was struggling with the decision. If I split the blob and OCI policy commands, there will be a lot of similar repeated code. If I merge them, there will be many if-else statements. Now, I have decided to give up on abstracting it. Let's create separate code for them.